### DISCUSSION TITLE
Infos pour collecte des Données

### DISCUSSION THREAD

**Discussion by salomebsk** [BIO6032/2022-SEIR-reproduction#34] [General]
from 1 September 2020 to 10 December 2020, or 101 d (tiré de l'article)

**Comment by blanchette31** [comment]
La license sur les données est celle de paternité de Creative Commons 3.0 et donc, la moins restrictive. Et on peut les utiliser si on crédite l'auteur (ce qu'on fera évidemment).

**Comment by blanchette31** [comment]
Population Suffolk County 2020 : 2020 US Census https://www.census.gov/library/visualizations/interactive/2020-population-and-housing-state-data.html

**Reply by salomebsk** [reply]
" **_Je ne suis pas certain de la population de Suffolk County qu'ils ont utilisés pour ajuster les données sur une population de 10 000 personnes._** "

Ici on peut voir qu'ils estiment qu'en 2020 il y avait une densité de population de 13 697 personnes par mile carré (voir l'encadré pour Suffolk County, MA ci-dessous issu du lien que tu as posté https://www.census.gov/library/visualizations/interactive/2020-population-and-housing-state-data.html )
Peut-être qu'ils ont considéré que l'université et 'ses alentours' (la source d'exogeneous input) fait un peu moins d'1 mile^2 et donc ils ont arrondi à 10 000 personnes.

Vous en dites quoi ? Possible ?

![image](https://user-images.githubusercontent.com/82335200/161834519-0dc7ff1d-53b4-4c10-a265-0261feef7c4f.png)

**Reply by salomebsk** [reply]
l'image devrait être plus claire ici:
![image](https://user-images.githubusercontent.com/82335200/161835567-390c8fcc-a6d3-422c-bb45-7e7f53900da8.png)

**Reply by blanchette31** [reply]
Non ils ont pris la taille de la population du campus de Boston University directement. Ils n'ont pas eu à l'estimer avec la densité de la population de Suffolk County ils le savent c'est quoi la population Brown travaille à cette Université là donc il doit juste avoir trouvé ou demandé le nombre d'étudiants et employés. Ce que je voulais dire pour la taille de population qu'ils ont utilisés pour arrondir à 10 000 c'est qu'on sait que la population N finale qu'on veut est de 10 000 et on sait le nombre de cas par jour qu'il a eu a Suffolk County dans la période que roule le modèle mais on ne savait pas la taille de la population de Suffolk County qu'il utilisait pour obtenir son nombre de cas ajusté à une population de 10 000.

**Reply by blanchette31** [reply]
Et c'est ça comme on peut voir dans le census de 2020 c'est environ 800 000 donc j'ai pris le nombre de nouveaux cas à chaque jour et j'ai divisé par 797936 et multiplié pour avoir une approximation du nombre de cas que ça serait environ si la taille de la population de Suffolk County était la même que celle de l'Université. C'est juste pour avoir des chiffres comparables pour la figure dans le fond.

**Reply by salomebsk** [reply]
ok ok je comprend ce que tu veux dire
et ça donne des résultats qui ont du sens avec ce qu'on observe sur les figures ?

**Reply by blanchette31** [reply]
vite vite ça semble bien la moyenne semble tourner autour de 3-4 cas par jour comme qu'il avait utilisé comme arrondissement pour les figures 2a-2b donc on est dans le même range. Je crois qu'il décide d'utiliser 101 jours au lieu de 100 parce que le gouvernement n'ont pas sortis les données journalières à l'action de grâce américaine (26 novembre) ils les ont juste ajoutées au données du 27 novembre donc on aura à probablement sauter le 26 novembre dans notre figure ou diviser les cas du 27 sur les deux jours. Il ne spécifie pas ce qu'il a décidé de faire donc je crois que ce sera à nous de voir.
